### ISSUE TITLE
Connector-scene?

### ISSUE THREAD

**Issue by nschneid** [UniversalConceptualCognitiveAnnotation/docs#56]
In the STREUSLE data I see annotations like "Everyone on staff is very professional and friendly", where "professional and friendly" is considered a complex state: 

is_F very_D [professional_C and_N friendly_C]_S

[full parse](https://github.com/danielhers/UCCA_English-EWT/blob/master-images/067423-0004.svg)

Is this correct? I don't see an example like this under the description of connectors; I thought those were only for use within non-S/P units.

**Comment by nschneid** [comment]
Contrast [this one](https://github.com/danielhers/UCCA_English-EWT/blob/master-images/083849-0002.svg), which uses parallel scenes, but seems to be missing a remote edge in the second scene.

**Comment by dotdv** [comment]
We've added a new category, Coordinated Main Relation (CMR). This is what we ask annotators: 
If two or more coordinated Processes or two or more coordinated States share exactly the same scene elements (As, Ds, Ts), then we unite them under one parent unit which is assigned two categories: Coordinated Main Relation (CMR) and also P or S accordingly. Internally, we use multiple Cs to mark each Main Relation separately. 
so: John [wrote_C and_N recorded_C]_P+CMR a song on Tuesday
On the other hand, if the coordinated Main Relations don't share one of the Scene elements, we separate them into different scenes.
[John_A [woke up]_P]_A,]_H [went_P [to school]_A (John)_A]_H and [met_P Mary_A there_A (John)]_H

The category was removed from the data for now (it appears in the annotated texts ). To my understanding they will add it back after they apply the expansion automatically. 

Re the missing Remote in [this example](https://github.com/danielhers/UCCA_English-EWT/blob/master-images/083849-0002.svg); that's a mistake.
